core/src/main/java/com/sequenceiq/cloudbreak/controller/StackCreatorService.java

```diff
@@ -400,8 +400,7 @@ private Blueprint determineBlueprint(StackV4Request stackRequest, Workspace work
             try {
                 boolean base = blueprintService.isAmbariBlueprint(blueprint) ? shouldUseBaseAmbariImage(clusterRequest) : shouldUseBaseCMImage(clusterRequest);
                 LOGGER.info("The stack with name {} will use base image: {}", stackName, base);
-                return imageService.determineImageFromCatalog(workspace.getId(), stackRequest.getImage(), platformString, blueprint, base, user,
-                        Optional.empty());
+                return imageService.determineImageFromCatalog(workspace.getId(), stackRequest.getImage(), platformString, blueprint, base, user, image -> true);
```

Comment:
just a minor thing: `Predicates.alwaysTrue()` could be better for reading the code

Reply:
Yes, originally I used it but IntelliJ suggested that it can be replaced with such a simple expression which might makes sense to not depend on external libraries, but since this is already the case I can put it back.